spec/spec_helper_acceptance.rb

```diff
@@ -0,0 +1,48 @@
+require 'beaker-rspec'
+
+# Various workarounds for Beaker's lack of Puppet 4 support.
+
```

Comment:
Yeah, you have to set a version over 4.0 in the environment variable PUPPET_INSTALL_VERSION, then it should test against newer Puppet.